modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/resource/v1_0/AccountResourceImpl.java

```diff
@@ -514,6 +512,8 @@ private AccountEntry _updateAccount(Long id, Account account)
 				accountEntry.getPrimaryKey(), customFields);
 		}
 
+		accountEntry = _accountEntryService.updateAccountEntry(accountEntry);
+
```

Comment:
I don't think this is needed because the `ExpandoUtil.updateExpando` should be not related to the model we are updating. I mean... it should only use the id for the relationship

Reply:
no ok, I got it... if we call the updateAccountEntry before it triggers the object action too early